src/operator/combineLatest.ts

```diff
@@ -31,3 +31,28 @@ export function combineLatest<T, R>(...observables: Array<Observable<any> |
 
   return new ArrayObservable(observables).lift<T, R>(new CombineLatestOperator<T, R>(project));
 }
+
+/* These types would be come standard types used through-out the library */
```

Comment:
Means this might need to be exported into different module later in common usage?